### PR TITLE
[Wiki] Added prerequisites section

### PR DESCRIPTION
**Pull request by AlexanderSehr**
# Description

- Added prerequisites section to wiki

# Type of Change

Please delete options that are not relevant.

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Update to documentation

### CODE REVIEW

File: docs/wiki/Getting started - Prerequisites.md
Comment (eriqua) [inline docs/wiki/Getting started - Prerequisites.md:1]: Please also add link to the wiki home page

File: docs/wiki/Getting started - Prerequisites.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+# Helper scripts
+
+This sub-section provides you with several helper scripts that can help you set your environment up.
+
+<details>
+<summary>Check versions script</summary>
+
+```PowerShell
+az --version
```
Comment (eriqua) [inline docs/wiki/Getting started - Prerequisites.md:48]:
```suggestion
az --version
az bicep --version
```

File: docs/wiki/Getting started - Prerequisites.md
Comment (AlexanderSehr) [reply docs/wiki/Getting started - Prerequisites.md:1]: done :)

**Review by eriqua — APPROVED** [review]
🆗

**Pull request merged**